### PR TITLE
Add project size limit in project settings 

### PR DESCRIPTION
**Pull request by imsahil007**
#8187 
Screenshot from 2020-07-06 22-40-20
Which file I have to change for updating the values inside these parameters?

### CODE REVIEW

File: warehouse/templates/manage/settings.html [hunk 1]
```diff
@@ -20,6 +20,12 @@
 {% block main %}
   <h2>{% trans %}Project settings{% endtrans %}</h2>
 
+  {% trans project_upload_limit=project.upload_limit %}Project upload limit is: 
+  <a href="https://pypi.org/help/#file-size-limit">{{ project_upload_limit }}{% endtrans %}</a><br><br>
+  
+  {% trans project_total_size_limit=project.total_size_limit %}Project total size limit is: 
+  <a href="https://pypi.org/help/#project-size-limit">{{ project_total_size_limit }}{% endtrans %}</a><br>
+
```
Comment (di) [inline warehouse/templates/manage/settings.html:28]: The actual values should come from the `project` variable which is available in this template (see how it's used elsewhere in this file)

The help links should be created with `request.help_url` instead.

**Comment by imsahil007** [comment]
Screenshot from 2020-07-07 01-58-15
I think this is okay. Followed both your advice and made the necessary changes.

**Comment by imsahil007** [comment]
I think this is okay. Followed both your advice and made the necessary changes.
Screenshot from 2020-07-07 02-24-33

File: warehouse/templates/manage/settings.html [hunk 2]
```diff
@@ -19,7 +19,10 @@
 
 {% block main %}
   <h2>{% trans %}Project settings{% endtrans %}</h2>
-
+  {% trans project_upload_limit=project.upload_limit | int / 1048576 , help_url=request.help_url(_anchor='file-size-limit') %}Project upload limit is: 
+  <a href="{{ help_url }}">{{ project_upload_limit }} MiB</a>{% endtrans %}<br><br>
+  {% trans project_total_limit=project.total_size_limit | int / 1073741824 , help_url=request.help_url(_anchor='project-size-limit') %}Project total size limit is: 
+  <a href="{{ help_url }}">{{ project_total_limit }} GiB</a>{% endtrans %}<br><br>
```
Comment (di) [inline warehouse/templates/manage/settings.html:25]: You can use the [`filesizeformat`](https://jinja.palletsprojects.com/en/2.11.x/templates/#filesizeformat) filter here instead to convert these to human-readable values.

Comment (imsahil007) [reply warehouse/templates/manage/settings.html:25]: Okay. I will use that. One more thing
Do I have to run 
$make translation 
Before committing?
It's not passing the test case.

Comment (di) [reply warehouse/templates/manage/settings.html:25]: Yes, see https://warehouse.readthedocs.io/development/submitting-patches/#translations

Comment (imsahil007) [reply warehouse/templates/manage/settings.html:25]: Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/sahil/Downloads/warehouse/warehouse/i18n/__init__.py", line 15, in <module>
    from babel.core import Locale
ModuleNotFoundError: No module named 'babel'
/.state/env/bin/isort -rc *.py warehouse/ tests/
/bin/sh: 1: /.state/env/bin/isort: not found
make: *** [Makefile:103: reformat] Error 127

I have already installed babel. I don't have any idea why this is happening.

**Pull request closed without merging**